### DISCUSSION TITLE
Nearly all of exportdm is "No messages found for the specified period."

### DISCUSSION THREAD

**Discussion by calebd-anderson** [Tyrrrz/DiscordChatExporter#687] [Q&A]
Only one of my discord DM channels is exporting. All the others say "`No messages found for the specified period.`" Even though clearly there are DMs there.

**Comment by Tyrrrz** [comment]
Did you specify a date range when exporting?

**Reply by calebd-anderson** [reply]
I attempted w/ `--after` flag and w/o `--after` flag. I should have specified in my original post it was the CLI that had the described behavior (or maybe that was implied). Update: when I used the GUI I was able to extract what I expected.
